Tools/cases_generator/test_generator.py

```diff
@@ -321,13 +321,13 @@ def test_macro_instruction():
                 _tmp_3 = res;
             }
             next_instr += 5;
-            static_assert(INLINE_CACHE_ENTRIES_OP == 5, "incorrect cache size");
```

Comment:
It isn't clear to me how/where the INLINE_CACHE_ENTRIES_OP instruciton is generated. 

Changing the test to pass w/o understanding doesn't inspire confidence on my end so any guidance or explanation here is greatly appreciated.

Reply:
It is the second parameter to `family`.